web-admin/tests/setup/setup.ts

```diff
@@ -208,7 +222,11 @@ setup.describe("global setup", () => {
           });
           return listing.textContent();
         },
-        { intervals: Array(6).fill(5_000), timeout: 30_000 },
+        {
+          // Increased timeout for the 1st dashboard to make sure sources are reconciled.
+          intervals: [10_000, 10_000, 20_000, 20_000, 30_000, 30_000],
+          timeout: 120_000,
+        },
```

Comment:
I'm nervous about this part. Are you sure this is needed? Given the test project is so small, it's a problem if the project can't be reconciled & ingested in 30s.

Reply:
It times out locally for me. IMO 2 mins is not a long time considering tests take about 10mins total. Other tests dont run if setup fails so I think it is a good compromise to have.

Reply:
Alright, let's try it